### ISSUE TITLE
Handle dates before 1900

### ISSUE THREAD

**Issue by ybkuang** [mapillary/mapillary_issues#589]
http://www.mapillary.com/map/im/0Q6_UwWeZPBlYWv2QUo-fw

**Comment by gyllen** [comment]
This is tricky!

**Comment by gyllen** [comment]
Do we handle these dates?

**Comment by jesolem** [comment]
@peterneubauer used negative epoch times for some cases. Works here?

**Comment by ybkuang** [comment]
Just check that it is doable. will do a fix soon.

**Comment by peterneubauer** [comment]
Yes, @neogeografen got in http://www.mapillary.com/map/im/no-fsWDnIx9dZRMO7T-IqA/photo yesterday.

**Comment by Neogeografen** [comment]
Hi I did use Geosetter for my 1932 pics yesterday - Can you give me an image before 1900 with lat/long and heading/direction and year takken, I will do a Geosetter EXIF test to see if it can handle dates before 1900 - 

One more old example from me is from 1902 http://www.mapillary.com/map/im/UdDwfrdhLwqPhrRkel4njg/photo

**Comment by ybkuang** [comment]
@Neogeografen I think geosetter is good with dates before 1900. It is just our backend that needs a fix.
